tests/integration/dev.bats

```diff
@@ -144,3 +152,26 @@ function teardown() {
 	runc exec -t test_exec sh -c "ls -l /proc/self/fd/0; echo 123"
 	[ "$status" -eq 0 ]
 }
+
+@test "runc run [hostname domainname]" {
+	requires root
+
+	setup_debian
```

Comment:
I used it because `busybox` doesn't have `domainname` command. If you have a good idea of how to get `domainname` with `busybox`, may I ask you to tell me?

Reply:
You can probably just `cat /proc/sys/kernel/domainname`?

Reply:
Good idea, Thanks @cyphar